install-dev/upgrade/sql/1.7.7.0.sql

```diff
@@ -12,3 +12,6 @@ ALTER TABLE `PREFIX_supply_order_detail` ADD `mpn` VARCHAR(40) NULL AFTER `upc`;
 ALTER TABLE `PREFIX_stock` ADD `mpn` VARCHAR(40) NULL AFTER `upc`;
 ALTER TABLE `PREFIX_product_attribute` ADD `mpn` VARCHAR(40) NULL AFTER `upc`;
 ALTER TABLE `PREFIX_product` ADD `mpn` VARCHAR(40) NULL AFTER `upc`;
+
+/* Set optin field value to 0 in employee table */
+UPDATE `PREFIX_employee` SET `optin` = 0;
```

Comment:
You also need to change the `NOT NULL` or add the default value to 0 to be consistent

Reply:
The same !

Reply:
Actually I removed the column from `install-dev/data/db_structure.sql` , we decided to let the column for people who upgrade (but set the value to zero for all), but to remove it completely for new installations.

Reply:
You guys are right, I missed your point, new added elements will have the default value for this field, which is 1.

Reply:
done !